specreduce/utils/utils.py

```diff
@@ -0,0 +1,209 @@
+import numpy as np
+
+from specreduce.core import _ImageParser
+from specreduce.tracing import Trace, FlatTrace
+from specreduce.extract import _ap_weight_image, _align_along_trace
+
+__all__ = ['measure_cross_dispersion_profile', '_align_along_trace']
+
+
+def measure_cross_dispersion_profile(image, trace=None, crossdisp_axis=0,
+                                     width=None, pixel=None, pixel_range=None,
+                                     statistic='median', align_along_trace=False):
```

Comment:
Sounds like the consensus is to change the default to `align_along_trace=True` (API docstring also needs updating)
```suggestion
                                     statistic='median', align_along_trace=True):
```

Reply:
Done